pipelines/cellpainting/cpd_analysis_pipeline.wdl

```diff
@@ -54,6 +57,7 @@ workflow cpd_analysis_pipeline {
     call util.cellprofiler_pipeline_task as cellprofiler {
       input:
         all_images_files = sp.array_output,
+        cppipe_file =cppipe_file,
```

Comment:
Just out of curiosity, why not leave this out and just let the `util.cellprofiler_pipeline_task` require the input?

Reply:
Good question. 
I tried, but when I created the meta-workflow, I couldn't pass inputs to tasks inside the workflow I was calling, they needed to be inputs tot he workflow. I could have been doing it wrong.
[Here](https://github.com/broadinstitute/cellprofiler-on-Terra/pull/53/commits/da5ac34a32899d6ef4297465eb8c1c78b0a369b5) you can see an example of how I tried and womtool labeled it as incorrect.

Reply:
Oh okay, I see what you tried.

But, for that example, what about just trying
```wdl
call cytomining_workflow.cytomining as cytomining
```
with no inputs specified at all?

Reply:
I still want to give some inputs, although I don't need all of them. 
In the `cytomining` example, I was trying to plug the output of the `cpd_analysis` pipeline as the input to `cytomining` and it wasn't allowing me. Is there a better/different way to do it?
Also `plate_id`, I wanted to be the same than the input for `create_load_data` so we didn't need to define them twice while running the test.
The other 2 could have been left unspecified.

Reply:
So, in this case, cppipe_file could have been left out, the motivation to add it here was to give it a more self-explanatory name, since otherwise, all (3) .cppipe are requested by util.cellprofiler_pipeline_task

Reply:
Oh okay yeah that makes some sense.  I guess that if you do not include them as explicit inputs, then they would be referred to as
```
cellpainting_workflow.cpd_analysis.cppipe_file
cellpainting_workflow.cp_illumination.cppipe_file
cellpainting_workflow.cpd_max_projection.cppipe_file
```
but if you do it as you have it, they are referred to as
```
cellpainting_workflow.analysis_cppipe_file
cellpainting_workflow.illumination_cppipe_file
cellpainting_workflow.max_projection_cppipe_file
```

I'm probably talking about this too much :)
We can definitely do it either way.  It's just if we change an input name or something in the future, it will have to be updated in multiple places.  But really it's totally fine this way.


And yes, you're right, for the example of the `cytomining` call, you do have to put 
```wdl
call cytomining_workflow.cytomining as cytomining {
  input:
    cellprofiler_analysis_directory_gsurl = cpd_analysis.analysis_output_directory,
}
```
at bare minimum.  You cannot leave it blank as I suggested above.

Reply:
ohhh, I see! 
You are right. Minimizing in how many places we need to update names, in case in case of a variable changing its name, it is indeed a good reason!